ccp/src/hashrate/prometheus.rs

```diff
@@ -39,6 +39,10 @@ async fn handler_404() -> impl response::IntoResponse {
     (http::StatusCode::NOT_FOUND, "No such endpoint")
 }
 
+async fn health(_: State<PrometheusMetrics>) -> impl response::IntoResponse {
+    (http::StatusCode::OK, "OK")
```

Comment:
Let's leave a comment on why that's a valid healthcheck strategy? Or todo if it's not yet implemented.